nixos/modules/services/misc/n8n.nix

```
@@ -0,0 +1,78 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.n8n;
+  format = pkgs.formats.json {};
+  configFile = format.generate "n8n.json" cfg.settings;
+in
+{
+  options.services.n8n = {
+
+    enable = mkEnableOption "n8n server.";
```

```suggestion
    enable = mkEnableOption "n8n server";
```

Period becomes redundant with `mkEnableOption`.

---

This is still unfixed @freezeboy .

---

Oops, fixing immediately